cardano-node/ChangeLog.md

```diff
@@ -1,6 +1,6 @@
 # Changelog for cardano-node
 
-## vNext
+## 1.35.5 -- November 2022
```

Comment:
Are we releasing from `master`? @disassembler

Reply:
this release should be 1.36 and released from master. There will be a 1.35.5 that's a minimal P2P release going out hopefully soon. @coot is working on that one, but we can back-port changelog from that release to master.

Reply:
Should I just rename this section to 1.36.0? 
I also noticed that the ledger and network sections don't have any entries. Should I just copy the changelog entries from these repos?